desktop/bip32ed25519/src/main/kotlin/bip32ed25519/utils.kt

```diff
@@ -0,0 +1,19 @@
+package bip32ed25519
+
+import net.pwall.json.schema.JSONSchema
+
+enum class KeyContext(val value: Int) {
+    Address(0),
+    Identity(1),
+}
+
+enum class Encoding {
+    CBOR,
```

Comment:
if we add CBOR we need to also add it to TS and test it

Reply:
CBOR was there before this PR, I just broke out the encoding enum into utils.kt.
Would you want me to erase CBOR from the Kotlin implementation?

Reply:
we can erase or comment out and add a comment that we'll add in the future

Reply:
Done